--- class/Common/Plugin/Assets.php	Fri Dec 21 10:45:21 2018
+++ class/Common/Plugin/Assets-patch.php	Fri Dec 21 10:46:04 2018
@@ -42,7 +42,7 @@
 		// add our custom CSS classes to <body>
 		add_filter( 'admin_body_class', array( $this, 'admin_body_class' ) );
 
-		$plugins_url = trailingslashit( plugins_url( $this->props->plugin_folder_name ) );
+		$plugins_url = trailingslashit( plugins_url( '/', $this->props->template_dir ) );
 		$version     = \defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? time() : $this->props->plugin_version;
 		$ver_string  = '-' . str_replace( '.', '', $this->props->plugin_version );
 
